superset-frontend/src/explore/components/Control.tsx

```diff
@@ -43,53 +43,38 @@ export type ControlProps = {
   renderTrigger?: boolean;
 };
 
-export default class Control extends React.PureComponent<
```

Comment:
Refactor from class component to functional. Not related to this PR but may be useful in the future.